Modules/HR/Resources/views/mail/infra-team.blade.php

```diff
@@ -0,0 +1,18 @@
+<div>
+    <style>
+        .line {
+            line-height: 1px;
+        }
+    </style>
+    <p>Dear Infrasupport Team,</p>
+    <p>I hope this email finds you well. I am writing to request the creation of an email account for a new employee
+        {{ $data['name'] }} in our organization.
+    </p>
+    <p>If there are any specific forms or documentation required for this process, kindly let me know, and I will
+        promptly provide the necessary information.</p>
+    <br>
+    <p>Thank you for your prompt attention to this matter. Your assistance is greatly appreciated.
```

Comment:
and use the above class here as:
```suggestion
    <p class="margin-bottom>Thank you for your prompt attention to this matter. Your assistance is greatly appreciated.
```